src/collections/_documentation/performance-monitoring/configuration/javascript.md

```diff
@@ -175,3 +175,88 @@ function processItem(item, transaction) {
   });
 }
 ```
+
+**Grouping Transactions**
```

Comment:
@markstory I moved your section into the JS Getting Started. Let me know if there are any inaccuracies. 🌻 Thanks!